### PR TITLE
Backend cleanup merge in

### PR DESCRIPTION
**Pull request by johnnicholson**

### CODE REVIEW

File: webServer/src/main/java/com/jnicho/quickpick/controllers/UserController.java [hunk 1]
```diff
@@ -119,35 +118,37 @@ public void createUser(@RequestBody UserRegistration registration) {
                                 .withValue(user.getFirstName() + " " + user.getLastName()))
                 .withForceAliasCreation(Boolean.FALSE);
 
-        AdminCreateUserResult result =  cognitoIdProvider.adminCreateUser(cognitoRequest);
-
+        AdminCreateUserResult result = cognitoIdProvider.adminCreateUser(cognitoRequest);
+        //TODO: Check result before saving
```
Comment (johnnicholson) [inline webServer/src/main/java/com/jnicho/quickpick/controllers/UserController.java]: INFO Complete the task associated to this TODO comment. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)

**Comment by johnnicholson** [comment]
SonarQube analysis reported 1 issue
* ![INFO][INFO] 1 info

Watch the comments in this conversation to review them.

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'

File: webServer/src/main/java/com/jnicho/quickpick/controllers/UserController.java [hunk 2]
```diff
@@ -221,7 +220,7 @@ public void confirmUser(@RequestBody UserConfirmation confirmation) {
 
     /**
      * an empty method that will error if the user is not authed
-     *
+     * <p>
```
Comment (thomassimko) [inline webServer/src/main/java/com/jnicho/quickpick/controllers/UserController.java]: this is weird

Comment (johnnicholson) [reply webServer/src/main/java/com/jnicho/quickpick/controllers/UserController.java]: uhhh... format tool???
not sure, I'll fix

**Pull request merged**